src/main/java/core/basesyntax/interfaces/Describer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface Describer {
```

Comment:
Interface names should not be verbs. Consider renaming 'Describer' to something that represents an entity or a property like 'DescriptionProvider' to clarify its purpose.